package.json

```diff
@@ -42,11 +42,13 @@
     "args": "^5.0.0",
     "args-parser": "^1.1.0",
     "bn.js": "^4.11.8",
+    "bytecode-utils": "^1.1.0",
     "consul": "^0.34.0",
     "ethjs-abi": "^0.2.1",
     "ethrpc": "^5.5.5",
     "request-promise": "^4.2.2",
     "sha256": "^0.2.0",
+    "truffle-flattener": "github:ceseale/truffle-flattener#master",
```

Comment:
@myhndl apart of my fix for the issue mentioned yesterday comparing bytecode required me to fix a bug in  `truffle-flattener`. Can you clone fork the repo so I can point this dependency to our org?

Reply:
Done: https://github.com/polyswarm/truffle-flattener

Please file a ticket to upstream this fix.